analysis_driver/pipelines/projects.py

```diff
@@ -39,6 +52,34 @@ def build_pipeline(dataset):
     return cleanup
 
 
+class CombineGVCFs(segmentation.Stage):
+    gvcfs = segmentation.ListParameter()
+    output_base = segmentation.Parameter()
+    reference = segmentation.Parameter()
```

Comment:
Is the `reference` parameter required since `CombineGVCFs` has access to `dataset`?

Reply:
Yes, looks like we can just reference `self.dataset.reference_genome`.

In that case, we can also remove the parameter from GenotypeGVCFs - unless we want to align to a different genome in certain cases, for some reason? Can't imagine us doing that at the moment, though.